kits/daisy/src/robot/hexapod.hpp

```diff
@@ -121,6 +123,9 @@ class Hexapod
 
   Eigen::Vector3d vel_xyz_;
 
+  // The orientation of gravity, as a unit vector, w.r.t. the chassis.
+  Eigen::Vector3d gravity_direction_;
```

Comment:
**Explicitly comment that this variable is not accessed or mutated in a thread safe manner**